### DISCUSSION TITLE
Can I integrate any LaTeX templates into a quarto file?

### DISCUSSION THREAD

**Discussion by zavud** [quarto-dev/quarto-cli#5438] [Q&A]
Is there a step-by-step guide of how to integrate scientific journal's templates (e.g. LaTeX) into quarto and write papers using a .qmd file?

I also found [this stackoverflow post](https://stackoverflow.com/questions/76111257/how-to-integrate-a-specific-journal-template-in-a-quarto-file) asking pretty much the same thing.

**Comment by mcanouil** [comment]
See journal articles extension and the related documentation on how a create a template extension: <https://quarto.org/docs/extensions/listing-journals.html>.

Also see partials to replace part of whole template used in Quarto: <https://quarto.org/docs/journals/templates.html>.
